bin/fcomm-index-packages

```diff
@@ -75,6 +75,8 @@ if __name__=="__main__":
             log.warn("No icons_dest specified.  Skipping copy.")
         else:
             icon_dir = os.path.join(options.cache_path, 'icons')
+            if not os.path.isdir(icon_dir):
+                icon_dir = os.path.join(options.cache_path, '..', 'icons')
```

Comment:
Heh, we could.. but it fails on the next line anyways (which is what prompted me to add this..)